cmd/plugin.go

```diff
@@ -358,7 +363,11 @@ func writeClassadOutputAndBail(exitCode int, resultAds []*classads.ClassAd) {
 	}
 
 	// We'll exit 3 in here if anything fails to write the file
-	writeOutfile(resultAds, outputFile)
+	_, retryable := writeOutfile(resultAds, outputFile)
+
+	if retryable {
```

Comment:
Hm - this seems brittle.  If we accidentally set retryable to true on success, it'll make the exit code 11, right?

Reply:
This function is used only when we need to bail early (such as with a panic or failing to InitClient). Therefore, we should only ever be failing here so success should never be true. Our exit codes out of this function should be either exit(1) or exit(11). I at least can't think of a case where we'd be successful but end up within this function but correct me if I'm wrong with any of this.

Reply:
Makes sense, thanks.